corehq/apps/analytics/tasks.py

```diff
@@ -368,7 +369,8 @@ def track_periodic_data():
     submit = []
     for user in users_to_domains:
         email = user.get('email')
-        if not email:
+        if not email or not validate_email(email, verify=True):
```

Comment:
there's also this email lib that checks the domain name: https://github.com/JoshData/python-email-validator. overall it seems like a better library. i might go with that